sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -8875,17 +8875,8 @@ pi_result piextPeerAccessGetInfo(pi_device command_device,
                                  size_t param_value_size, void *param_value,
                                  size_t *param_value_size_ret) {
 
-  std::ignore = command_device;
-  std::ignore = peer_device;
-  std::ignore = attr;
-  std::ignore = param_value_size;
-  std::ignore = param_value;
-  std::ignore = param_value_size_ret;
-
-  setErrorMessage("piextPeerAccessGetInfo not "
-                  "implemented in L0",
-                  UR_EXT_RESULT_ADAPTER_SPECIFIC_ERROR);
-  return PI_ERROR_PLUGIN_SPECIFIC_ERROR;
+  ReturnHelper ReturnValue(ParamValueSize, ParamValue, ParamValueSizeRet);
+  return ReturnValue(0);
```

Comment:
Just for good measure, could we add a small comment about this just reporting 0 to signify that all of the queries report false, i.e. unsupported, for the backends where we do this?